.github/workflows/test.yaml

```diff
@@ -42,8 +42,18 @@ jobs:
       fail-fast: true
       matrix:
         python_version: ['3.8', '3.10']
-        method: [linear, robust, nroot, selective, auto_covariance, pws]
-        freq_norm: [rma, no, phase_only]
+        method: [linear]
+        freq_norm: [rma]
+        format: [asdf]
+        include:
+        - method: [robust, nroot, selective, auto_covariance, pws]
+          python_version: ['3.8', '3.10']
+          freq_norm: [no]
+          format: [zarr]
+        - method: [robust]
+          python_version: ['3.8', '3.10']
```

Comment:
Any reason why Python 3.9 is not included in the testing?

Reply:
Just to keep the matrix smaller since I was doing the full matrix testing. I opted for testing the min and max of supported versions. But actually now that I discovered the `include` argument I can add `3.9` for a single configuration as a good sanity check.

Reply:
Added one test with 3.9